src/main/java/com/rybka/configuration/ApplicationConfiguration.java

```diff
@@ -17,8 +18,7 @@
 
 @Configuration
```

Comment:
Do you need this configurations? 
SpingBootApplication provides all of them. 
Take a look into the annotation/

Reply:
fixed